html/user/per_app_list.php

```diff
@@ -63,13 +67,20 @@ function show_header($is_team, $apps, $appid, $is_total) {
 // show a user or team, with their credit for each app
 //
 function show_row($item, $apps, $is_team, $i) {
```

Comment:
Rough sketch: Loop over the array returned by `get_top_items` and query credit and RAC per app and add this to `$data` (so it ends up in `$item` later). Then cache `$data`. Adjust the `foreach ($apps as $app) {` loop in `show_row` to not query the DB but look in `$item` for credit and RAC.